src/go/rdctl/pkg/snapshot/manager.go

```diff
@@ -48,6 +48,19 @@ func NewManager(paths paths.Paths) Manager {
 	}
 }
 
+func (manager *Manager) GetSnapshotId(desiredName string) (string, error) {
+	snapshots, err := manager.List()
+	if err != nil {
+		return "", fmt.Errorf("failed to list snapshots: %w", err)
+	}
+	for _, candidate := range snapshots {
+		if desiredName == candidate.Name {
+			return candidate.ID, nil
+		}
+	}
+	return "", fmt.Errorf(`can't find a snapshot with name %q`, desiredName)
```

Comment:
The "with name" part feels clunky; the name is the unique identifier for the snapshot; there is no other kind of handle for it:

```suggestion
	return "", fmt.Errorf(`can't find snapshot %q`, desiredName)
```